.actions/setup_tools.py

```diff
@@ -270,6 +270,37 @@ def prune_comments_docstrings(lines: List[str]) -> List[str]:
     return body
 
 
+def wrap_try_except(body: List[str], pkg: str, ver: str) -> List[str]:
+    """Wrap the file with try/except for better traceability of imp0rt misalignment."""
+    not_empty = sum(1 for ln in body if ln)
+    if not_empty == 0:
+        return body
+    body = ["try:"] + [f"    {ln}" if ln else "" for ln in body]
+    body += [
+        "",
+        "except ImportError as err:",
+        "",
+        "    from os import linesep",
+        f"    from {pkg} import __version__",
+        f"    msg = f'Your meta package was build for `{pkg}=={ver}`," + " but you are running {__version__}'",
```

Comment:
```suggestion
        f"    msg = f'Your meta package was built for `{pkg}=={ver}`," + " but you are running {__version__}'",
```

Reply:
I don't think a user knows what a meta package is (frankly, I don't even feel like I do). 
Isn't this just `lightning`?

Reply:
good point